src/hooks/screen/useChatScreenHook.ts

```diff
@@ -49,6 +50,7 @@ function useChatScreenHook(type: 'SIGNED' | 'ANONYMOUS'): UseChatScreenHook {
 
   const sendChat = async (
     message: string = randomString(20),
+    attachments: [],
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `sendChat` function now includes an `attachments` parameter. This change looks good as long as the callers of this function are updated accordingly to pass this new argument.
```diff
+     attachments: [],
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->